server/etcdmain/config.go

```diff
@@ -301,6 +301,7 @@ func newConfig() *config {
 	fs.DurationVar(&cfg.ec.ExperimentalWarningApplyDuration, "experimental-warning-apply-duration", cfg.ec.ExperimentalWarningApplyDuration, "Time duration after which a warning is generated if request takes more time.")
 	fs.BoolVar(&cfg.ec.ExperimentalMemoryMlock, "experimental-memory-mlock", cfg.ec.ExperimentalMemoryMlock, "Enable to enforce etcd pages (in particular bbolt) to stay in RAM.")
 	fs.BoolVar(&cfg.ec.ExperimentalTxnModeWriteWithSharedBuffer, "experimental-txn-mode-write-with-shared-buffer", true, "Enable the write transaction to use a shared buffer in its readonly check operations.")
+	fs.BoolVar(&cfg.ec.ExperimentalStopGRPCServiceOnDefrag, "experimental-stop-grpc-service-on-defrag", cfg.ec.ExperimentalStopGRPCServiceOnDefrag, "Enable etcd gRPC service to stop serving client requests on defragmentation.")
```

Comment:
Adding `experimental` prefix and removing it in future is a breaking change as we discussed in https://github.com/etcd-io/etcd/issues/17657. Also see https://github.com/etcd-io/etcd/issues/17657

Are we still happy to add prefix `experimental` for an flag?

I know this PR just backports changes from main to 3.5, so the suggestion (adding `experimental` in description only) is actually for the main branch (update main firstly). Do we want to follow the suggestion for now? Or do we have an agreement on https://github.com/etcd-io/etcd/issues/17657 and https://github.com/etcd-io/etcd/issues/17657 before we add any new flags? @fuweid @jmhbnz @serathius @siyuanfoundation @spzala 

```suggestion
	fs.BoolVar(&cfg.ec.ExperimentalStopGRPCServiceOnDefrag, "stop-grpc-service-on-defrag", cfg.ec.ExperimentalStopGRPCServiceOnDefrag, "(Experimental) Enable etcd gRPC service to stop serving client requests on defragmentation.")
```

Reply:
As part of https://github.com/etcd-io/etcd/issues/17657 we will still need to support migration from the old flag naming scheme (with prefix) to the new one. One more flag should not change much as we already have other flags to migrate. So I don't think we need to block on this.

Reply:
I'm happy for this to proceed using older approach. We are still working on the KEP for feature flags so it will be a while off and agree with @serathius above we don't want to block this work.